include/motor-control/core/motor_interrupt_handler.hpp

```diff
@@ -184,27 +198,27 @@ class MotorInterruptHandler {
         return bool((current ^ future) & overflow_flag);
     }
 
+    // test interface
     [[nodiscard]] auto get_current_position() const -> q31_31 {
         return position_tracker;
     }
-
     void set_current_position(q31_31 pos_tracker) {
         position_tracker = pos_tracker;
     }
-
+    bool has_active_move = false;
```

Comment:
Is this supposed to be here?

Reply:
Yeah that was that testing-only member. the diff on this file is because I manually copied it back to the header